### ISSUE TITLE
Dropdown lists crashes application when using fluxbox as window manager

### ISSUE THREAD

**Issue by thulle** [sadko4u/lsp-plugins#315]
Hi again, got another fluxbox related issue for you! :)

I can start applications just fine, for example the spectrum analyzer:
```

$ lsp-plugins-spectrum-analyzer-x1
Trying to connect to JACK
Successfully connected to JACK
```

Buttons & dials work great, but if i click any dropdown list to show the options, like Mode, Window, Tolerance and Envelope in the spectrum analyzer, it crashes with:

```
X Error of failed request:  BadMatch (invalid parameter attributes)
  Major opcode of failed request:  42 (X_SetInputFocus)
  Serial number of failed request:  10204
  Current serial number in output stream:  10206
```

Serial numbers change between runs, but it's always 42 (X_SetInputFocus).

Now a curious quirk is this: If i trace the application with strace (running it with "strace lsp-plugins-spectrum-analyzer-x1" from a terminal), it doesn't crash. Instead it has part of the window decoration issue from my earlier  #207 - it shows the decoration, but in this case the menu doesn't lose focus, so in this case the menu is still usable.

![2023-04-03-093025_7680x1440_scrot](https://user-images.githubusercontent.com/1105866/229441598-68437739-d2d3-4a1e-96e6-0db13c6a2ce8.png)

Maybe some kind of race condition since strace slowing it down is helping in some kind of way.

Since it's mentioning focus it might be worth mentioning that I use fluxbox with the "mouse focus" model + focus new windows, ie. windows only have focus when the mouse pointer is above them, but new windows that pop up will have focus regardless of where the mousepointer is. Both configurable under the Fluxbox Configuration -> Focus model right click menu.

Also, thanks a lot for your work on these apps, it's a great toolbox to have!

**Comment by sadko4u** [comment]
Hello!
UI uses only one single thread for message dispatching and drawing.
Can not actually say what's the problem, need to inspect and reproduce it first.

**Comment by sadko4u** [comment]
Fixed in 1.2.7
